tests/client/core/TestInstallAddon.js

```diff
@@ -96,6 +97,10 @@ describe('withInstallHelpers inner functions', () => {
     mapDispatchToProps = makeMapDispatchToProps({ WrappedComponent, src });
   });
 
+  beforeEach(() => {
+    configStub = sinon.stub(config, 'get').withArgs('server').returns(false);
```

Comment:
The sinon we use is actually a sandbox and that gets implicitly reset here https://github.com/mozilla/addons-frontend/pull/2413/files#diff-81e799b66ac20327dd8a1b8bb1f5f802R69